src/main/app/forms/validation/validators/isCountrySupported.ts

```diff
@@ -13,13 +13,22 @@ import { request } from 'client/request'
 
 const postcodeClient = new PostcodeInfoClient(config.get<string>('postcodeLookup.apiKey'), request)
 
+export enum BlockedPostcodes {
+  ISLE_OF_MAN = 'IM'
+}
+
 @ValidatorConstraint()
 export class CheckCountryConstraint implements ValidatorConstraintInterface {
 
   async validate (value: any | string, args?: ValidationArguments): Promise<boolean> {
     if (value === undefined || value === null || value === '') {
       return true
     }
+
+    if (value.toUpperCase().startsWith(BlockedPostcodes.ISLE_OF_MAN)) {
```

Comment:
Shall we trim the value before doing `startsWith`? I guess there is a chance to provide postcode that starts with ` IM` (whitespace prefixed).